packages/react-page-renderer/src/AbstractClientPageRenderer.ts

```diff
@@ -87,6 +88,7 @@ export default abstract class AbstractClientPageRenderer extends AbstractPageRen
           await this._renderPageViewToDOM(controller, pageView, routeOptions);
         }
 
+        controller.getMetaManager().clearMetaAttributes();
```

Comment:
Controller decorator's is used only for auto injection MetaManager, Router, etc to setMetaParams method. Controller is not owner of MetaManager. So controller.getMetaManager() is only bad short cuts for regular depdency in constructor. Maybe we should move updating meta information to PageHandlers.